internal/ycsb/ycsb.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package ycsb contains ycsb runner.
+package ycsb
+
+import (
+	"context"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/FerretDB/dance/internal"
+)
+
+// Run runs YCSB workloads.
+// It loads and runs a YCSB workload. Properties defined in the YAML file
+// will override properties defined in the workload parameter file.
+func Run(ctx context.Context, dir string, args []string) (*internal.TestResults, error) {
+	res := &internal.TestResults{
+		TestResults: make(map[string]internal.TestResult),
+	}
+
+	_, err := os.Stat("../bin/go-ycsb")
```

Comment:
Let's use `filepath.Join` everywhere to make it work on Windows

Reply:
Oh yes, complete oversight on my part.